src/components/CurrentWeather/CurrentWeather.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+
+function CurrentWeather({ description, temp_min, temp_max, humidity, pressure }) {
+  return (
+    <>
+      <div className="description">
+        <h2>{description}</h2>
+      </div>
+
+      <div className="temp">
+        <h3>
+          Temperature : {temp_min}° to {temp_max}°C
+        </h3>
+      </div>
+
+      <div className="box">
+        <div className="sub-box">
+          <h4>Humidity : {humidity}%</h4>
+        </div>
+
+        <div className="sub-box">
+          <h4>Pressure : {pressure}</h4>
+        </div>
+      </div>
```

Comment:
Do you need to wrap each heading inside a div? What is the intention here? Could you not put the CSS class (description, temp, sub-box etc) on the heading element itself? 
Remember we only want the minimum amount of markup to make a cohesive, semantic structure for our content.

Reply:
I see. I deleted some divs :) thank you